e2e-tests/README.md

```diff
@@ -142,8 +76,75 @@ command line options available. To see the options available, run
 __Troubleshooting__
 
 If the test run hangs with Firefox open but no URL gets entered in the address
-box, some combination of the Firefox version, and the python Selenium bindings
-version may not be compatible. Upgrading each of them to latest should fix it.
+box, some combinations of the Firefox version, and the python Selenium bindings
+version may not be compatible. Upgrading each of them to latest often fixes it.
+ 
+Tips and tricks
+---------------
+
+Because Selenium opens real browser windows, it will steal focus and switch
+workspaces. Firefox doesn't have a headless mode of operation, so we can't 
+simply turn off the UI.
+
+__Use a differnet driver__
+
+[pytest-selenium][pytest-selenium] provides the ability to run tests against other 
+driver executables as well as external providers -- execuatables and providers include PhantomJS, SauceLabs, BrowserStack, etc.
```

Comment:
Is there an eta on when that documentation will land? I'll gladly update the docs here to point to that.